src/dcd/server/autocomplete/complete.d

```diff
@@ -164,11 +201,21 @@ AutocompleteResponse dotCompletion(T)(T beforeTokens, const(Token)[] tokenArray,
 			cursorPosition, CompletionType.identifiers, false, partial);
 		break;
 	case tok!"(":
-	case tok!"{":
 	case tok!"[":
-	case tok!";":
 	case tok!":":
 		break;
+	//  these tokens before a "." means "Module Scope Operator"
+	case tok!"{":
+	case tok!";":
+	case tok!"}":
```

Comment:
wouldn't `[`, `(`, `,`, `:` before `.` also mean it's a module scope operator?
```d
int[] x = [.global];

fun(.global, .global2);

case Foo:
.global();
```

Reply:
yes

Reply:
done